config/routes.rb

```diff
@@ -1,10 +1,18 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  devise_for :users
+  devise_scope :user do
+    get '/users/sign_out' => 'devise/sessions#destroy'
```

Comment:
- [ ] Please kindly resolve these two failing tests, though I thought if you can resolve the general_shopping_lists error; I'm thinking that the tests might passed. Welldone. 🙏 

<img width="472" alt="Untitled" src="https://github.com/chimwemwe007/my-recipe-app/assets/103386183/8c828cf5-8f79-43cb-a13d-c84ad9a60c51">